WIP: updates to Connect docs (tx settings) (#218)
wormhole-foundation/wormhole-docs: build/applications/connect/configuration/configure-data.md

--8<-- 'code/build/applications/connect/configuration/custom-tokens-whitelist.jsx'
```

> You can whitelist tokens by symbol or by specifying tuples of [chain, address]. For example, this would show only BONK token (on all chains you've whitelisted) as well as [`EPjFW...TDt1v`](https://solscan.io/token/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v){target=\_blank} on Solana, which is USDC.

Reviewer: Line 115 needs some editing, this was the closest I could get to it but I just noticed it @ilariae 

"You can customize the tokens shown in the UI using the `tokens` property. The following example adds a custom token and limits Connect to showing only that token, along with the native gas tokens ETH and SOL."

```jsx
--8<-- 'code/build/applications/connect/configuration/custom-tokens-whitelist-advanced.jsx'
```

> ### User-inputted Tokens {: #user-inputted-tokens }

Reviewer: ```suggestion
### User-Inputted Tokens {: #user-inputted-tokens }
```

--8<-- 'code/build/applications/connect/configuration/custom-tokens-whitelist.jsx'
```

> You can whitelist tokens either by symbol by specifying tuples of [chain, address]. For example, this would show only BONK token (on all chains you've whitelisted)
> as well as [EPjFW...TDt1v](https://solscan.io/token/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v) on Solana, which is USDC.

Reviewer 2: ```suggestion
You can whitelist tokens by symbol or by specifying tuples of [chain, address]. For example, this would show only BONK token (on all chains you've whitelisted) as well as [`EPjFW...TDt1v`](https://solscan.io/token/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v){target=\_blank} on Solana, which is USDC.
```

```jsx
--8<-- 'code/build/applications/connect/configuration/custom-tokens-whitelist-advanced.jsx'
```

### User-inputted Tokens {: #user-inputted-tokens }

> As of version 2.0, Connect allows users to paste token addresses in to bridge any token they want. As an integrator, you may want to disable
> this feature if you are deploying Connect for use only with a specific token(s).

Reviewer 2: ```suggestion
As of version 2.0, Connect allows users to paste token addresses to bridge any token they want. As an integrator, you may want to disable this feature if you are deploying Connect for use only with a specific token(s).
```

If you provide a token whitelist (see above), this is turned off automatically. However, you can also disable it explicitly like this:

```jsx
--8<-- 'code/build/applications/connect/configuration/custom-disable-arbitrary-tokens.jsx'
```

Setting `ui.disableUserInputtedTokens` to `true` will disable the ability to paste in token addresses.

### Transaction Settings {: #transaction-settings }

> Landing transactions on Solana can require finely tuned priority fees, when there is congestion. You can tweak how Connect determines these with `transactionSettings`.
> All of the parameters in this config are optional; you can provide any combination of them.

Reviewer 2: ```suggestion
Landing transactions on Solana can require finely tuned priority fees when there is congestion. You can tweak how Connect determines these with `transactionSettings`. All of the parameters in this configuration are optional; you can provide any combination of them.
```

```jsx
--8<-- 'code/build/applications/connect/configuration/custom-tx-settings-solana.jsx'
```

!!! note

>     Connect can calculate fees more accurately if you are using a [Triton](https://triton.one) RPC endpoint.

Reviewer 2: ```suggestion
    Connect can calculate fees more accurately if you are using a [Triton](https://triton.one){target=\_blank} RPC endpoint.
```
